raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -0,0 +1,471 @@
+
+Offences relating to Environmental Impact Assessment
+(138) Any person who: [a] fails to submit a project report contrary to the requirements of section 58 of this Act; [b] fails to prepare an environmental impact assessment report in accordance with the requirements of this Act or regulations made thereunder; [c] fraudulently makes false statements in an environmental impact assessment report submitted under this Act or regulations made thereunder, commits an offence and is liable on conviction to imprisonment for a term not exceeding twenty four months or to a fine of not more than two million shillings or to both such imprisonment and fine.
+
+Offences relating to records
+(139) Any person who: [a] fails to keep records required to be kept under this Act; [b] fraudulently alters any records required to be kept under this Act; [c] fraudulently makes false statements in any records required to be kept under this Act, commits an offence and is liable upon conviction to a fine of not more than five hundred thousand shillings or to imprisonment for a term of not more than eighteen months or to both such fine and imprisonment.
+
+Offences relating to standards
+(140) Any person who: [a] contravenes any environmental standard prescribed under this Act; [b] contravenes any measure prescribed under this Act; [c] uses the environment or natural resources in a wasteful and destructive manner contrary to measures prescribed under this Act, commits an offence and shall be liable upon conviction, to a fine of not more than five hundred thousand shillings or to imprisonment for a term of not more than twenty-four months or to both such fine and imprisonment.
+
+Offences relating to hazardous wastes, materials, chemicals and
+(141) radioactive substances Any person who: [a] fails to manage any hazardous waste and materials in accordance with this Act; [b] imports any hazardous waste contrary to this Act; [c] knowingly mislabels any waste, pesticide, chemical, toxic substance or radioactive matter; [d] fails to manage any chemical or radioactive substance in accordance with this Act; [e] aids or abets illegal trafficking in hazardous waste, chemicals, toxic substances and pesticides or hazardous substances; [f] disposes of any chemical contrary to this Act or hazardous waster within Kenya; [g] withholds information or provides false information about the management of hazardous wastes, chemicals or radioactive substances, commits an offence and shall, on conviction, be liable to a fine of not less than one million shillings, or to imprisonment for a term of not less than two years, or to both.
```

Comment:
Please remove `radioactive substances` from the beginning of this article. Also, please replace `hazardous waster` with `hazardous waste`.